feature_aggregation_scripts/ocean_subBasin_regions/ocean_subBasin_regions.py

```diff
@@ -0,0 +1,244 @@
+    fcFlores = gf.read('ocean', 'region', ['Flores Sea'])
+    fcTimor = gf.read('ocean', 'region', ['Timor Sea'])
+    fcTasmanN = gf.read('ocean', 'region', ['Tasman Sea North'])
+    fcCoral = gf.read('ocean', 'region', ['Coral Sea'])
+    fcSP.merge(fcBali)
+    fcSP.merge(fcSavu)
+    fcSP.merge(fcMak)
+    fcSP.merge(fcAra)
+    fcSP.merge(fcCeram)
+    fcSP.merge(fcBis)
+    fcSP.merge(fcSolo)
+    fcSP.merge(fcMol)
+    fcSP.merge(fcBanda)
+    fcSP.merge(fcGBoni)
+    fcSP.merge(fcGTomini)
+    fcSP.merge(fcJava)
+    fcSP.merge(fcFlores)
+    fcSP.merge(fcTimor)
+    fcSP.merge(fcTasmanN)
+    fcSP.merge(fcCoral)
+    fcSP = fcSP.combine('South Pacific Basin')
+    props = fcSP.features[0]['properties']
+    props['tags'] = ['South_Pacific_Basin', 'oceanSubBasinRegions']
+    props['author'] = author
+
+    # Create Indian Ocean
+    fcI = gf.read('ocean', 'region', ['Indian Ocean'])
+    fcBassStN = gf.read('ocean', 'region', ['Bass Strait North'])
+    fcBayBengal = gf.read('ocean', 'region', ['Bay of Bengal'])
+    fcGulfOman = gf.read('ocean', 'region', ['Gulf of Oman'])
+    fcMalaccaSt = gf.read('ocean', 'region', ['Malacca Strait'])
+    fcGABN = gf.read('ocean', 'region', ['Great Australian Bight North'])
+    fcMoz = gf.read('ocean', 'region', ['Mozambique Channel'])
+    fcBu = gf.read('ocean', 'region', ['Andaman or Burma Sea'])
+    fcAden = gf.read('ocean', 'region', ['Gulf of Aden'])
+    fcLacc = gf.read('ocean', 'region', ['Laccadive Sea'])
+    fcArab = gf.read('ocean', 'region', ['Arabian Sea'])
+    fcI.merge(fcBassStN)
+    fcI.merge(fcBayBengal)
+    fcI.merge(fcGulfOman)
+    fcI.merge(fcMalaccaSt)
+    fcI.merge(fcGABN)
+    fcI.merge(fcMoz)
+    fcI.merge(fcBu)
+    fcI.merge(fcAden)
+    fcI.merge(fcLacc)
+    fcI.merge(fcArab)
+    fcI = fcI.combine('Indian Ocean Basin')
+    props = fcI.features[0]['properties']
+    props['tags'] = ['Indian_Ocean_Basin', 'oceanSubBasinRegions']
+    props['author'] = author
+
+    # Create Southern Ocean
+    fcSO = gf.read('ocean', 'region', ['Southern Ocean'])
+    fcTasmanS = gf.read('ocean', 'region', ['Tasman Sea South'])
+    fcBassStS = gf.read('ocean', 'region', ['Bass Strait South'])
+    fcGABS = gf.read('ocean', 'region', ['Great Australian Bight South'])
+    fcSO.merge(fcTasmanS)
+    fcSO.merge(fcBassStS)
+    fcSO.merge(fcGABS)
+    fcSO = fcSO.combine('Southern Ocean Basin')
+    props = fcSO.features[0]['properties']
+    props['tags'] = ['South_Ocean_Basin', 'oceanSubBasinRegions']
+    props['author'] = author
+
+    # Create Ocean subBasins merged feature
+    fc = fcSO
+    fc.merge(fcAr)
+    fc.merge(fcNA)
+    fc.merge(fcSA)
+    fc.merge(fcNP)
+    fc.merge(fcSP)
+    fc.merge(fcI)
+    props = fc.features[0]['properties']
+    props['tags'] = 'oceanSubBasinRegions'
+    props['author'] = author
+
+    fc.plot(projection='cyl')
+    fc.to_geojson('oceanSubBasins20201008.geojson')
+
+    plt.show()
```

Comment:
The calling code can plot and/or save `fc` but that shouldn't be part of this function in my opinion.

```suggestion
    return fc
```